ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -203,3 +203,50 @@ def record_usage(self, span: Span, usage: Dict[str, Any]) -> None:
         total_cost = span.get_metric(TOTAL_COST)
         if total_cost:
             self.metric(span, "incr", "tokens.total_cost", total_cost)
+
+    def tag_lcel_inputs(self, span: Span, inputs):
+        """
+        Tags the span with the inputs to the LCEL chain,
+        taking into account for the differences in structure (invocation, batching).
+        Also, puts these values into a dictionary for returning an object for logs.
+        """
+        if isinstance(inputs, list):
+            log_inputs: Dict[str, Any] = {str(idx): {} for idx in range(len(inputs))}
```

Comment:
If we don't need to return the `log_inputs` since we're not submitting logs, we can just remove this variable. wdyt?